topi/python/topi/nn/dilation2d.py

```diff
@@ -0,0 +1,130 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
merge it in dilate.py?

Reply:
merged in dilate.py, file deleted dilation2d.py